pydust/src/pydust.zig

```diff
@@ -120,6 +120,7 @@ fn eagerEval(comptime definition: type) void {
     }
     for (@typeInfo(definition).Struct.decls) |d| {
         const value = @field(definition, d.name);
+        @setEvalBranchQuota(2048);
```

Comment:
```suggestion
        @setEvalBranchQuota(10000);
```
Go all the way by order of magnitude? This only increases memory and time of compilation

Reply:
Sure